asciidoctorj-distribution/src/test/groovy/org/asciidoctor/diagram/WhenDitaaDiagramIsRendered.groovy

```diff
@@ -58,7 +59,10 @@ Hello World
 
 
         then:
-        result.contains("""src="${imagesOutDir.absolutePath}/${imageFileName}.png""")
+        if (Platform.getNativePlatform().OS == Platform.OS.WINDOWS)
+            result.contains("""src="${imagesOutDir.absolutePath.replaceAll("\\\\", "//")}/${imageFileName}.png""")
+        else
+            result.contains("""src="${imagesOutDir.absolutePath}/${imageFileName}.png""")
```

Comment:
I don't think we should be checking for absolute paths in the output. We would never want the HTML to point to an absolute path. But this is caused by the fact that we're passing an absolute path to imagesdir. I think imagesdir needs to be set to a relative path and then we don't have to worry about replacing any system path separators.